src/test/middleware/single_thread_app1/CMakeLists.txt

```diff
@@ -5,3 +5,8 @@ target_link_libraries(goby_test_single_thread_app1 goby_middleware goby_common d
 
 add_test(goby_test_single_thread_app1 ${goby_BIN_DIR}/goby_test_single_thread_app1)
 
+# Disable the test when building with ThreadSanitizer
+# https://github.com/GobySoft/goby3/issues/68
+if(CMAKE_CXX_FLAGS MATCHES "(^| )-fsanitize=thread( |$)")
```

Comment:
I opted to check `CMAKE_CXX_FLAGS` rather than `SANITIZE_THREAD` because the latter is specific to the cgsn-mooring project.